n3fit/src/n3fit/tests/test_evolven3fit.py

```diff
@@ -0,0 +1,72 @@
+import pathlib
+import logging
+from numpy.testing import assert_allclose
+import numpy as np
+import pytest 
+from validphys.pdfbases import PIDS_DICT
+from validphys.loader import Loader
+from evolven3fit_new import utils, eko_utils
+
+REGRESSION_FOLDER = pathlib.Path(__file__).with_name("regressions")
+log = logging.getLogger(__name__)
+
```

Comment:
To avoid the kind of errors I got (`evolven3fit_new` didn't run) it would be good to have some kind of test that is able to 1) start evolven3fit 2) load an eko (but not apply it because of the memory problem).

Just to make sure that it can at least run and load the eko from the theory. Something like a dry run.

Reply:
With this version of EKO, loading is already consuming enough memory...

Reply:
We can have a low-precision eko in the low-precision theory.

Reply:
But then, is there a memory problem in applying it?

Reply:
No, you are right. At that point one could probably finish applying and everything. No need for dry runs.

Reply:
Do we have a fake little fit that I can use to test the cli?

Reply:
No, for the test you already have a few fits. You can use the regression or the low-precision ones in the config file for the vp tests.